### PR TITLE
What to do if you have more than one version of Parity installed.

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: FAQ-Basic-Operations,-Configuration,-and-Synchronization.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ After building Parity from source:
 
 After installing Parity, just run `parity` from command line. See also [how do I configure Parity](https://github.com/paritytech/parity/wiki/FAQ#how-do-i-configure-parity) below.
 
+If you happen to have more than one version of Parity installed, and running `parity -v` gives an older version than the other version that is installed, you can run `which parity` to find where it is installed (it may be `/usr/bin/parity`, then `sudo cp <insert filepath of target/release/parity of the latest version> </usr/bin/parity or wherever the executable for parity is>, e.g. `sudo cp ~/paritytarget/release/parity /usr/bin/parity`.
```
Comment (Tbaut) [inline FAQ-Basic-Operations,-Configuration,-and-Synchronization.md]: Please correct the quotes

**Comment by 5chdn** [comment]
Thanks, but this is out of scope for the Parity documentation and not entirely valid for other operating systems that are not powered by GNU/Linux.

**Comment by jamesray1** [comment]
Is it out of scope? Don't you think that's a bit drastic? Instructions for other operating systems can also be included, this could be a starting point.

File: FAQ-Basic-Operations,-Configuration,-and-Synchronization.md (hunk 1 above)
Comment (jamesray1) [reply FAQ-Basic-Operations,-Configuration,-and-Synchronization.md]: It seems that I can't edit this, probably because it is closed.

**Comment by 5chdn** [comment]
You are describing how to repair a messed up Linux environment. This is out of scope because it has nothing to do with Parity and also the instructions are fundamentally wrong if not dangerous (sudo-copying files into your system-wide binary path).

**Pull request closed without merging**